kodilla-good-patterns/src/main/java/com/kodilla/good/patterns/food2door/Supplier.java

```diff
@@ -0,0 +1,9 @@
+package com.kodilla.good.patterns.food2door;
+
+import java.util.List;
+import java.util.Map;
+
+interface Supplier {
```

Comment:
format code! There is an option in Intellij that it will automatically format your code on every save/change.
Code indention are important and makes reading code easier.